cabby/data/extract.py

```diff
@@ -31,7 +31,7 @@
 
 def get_wikigeo_data(
     wikidata_items: Sequence[wdi.WikidataEntity]
-) -> Sequence[wikigeo.WikigeoEntity]:
```

Comment:
may be separate this out from this PR to avoid conflicts with the other in-prgress ones?

Reply:
should be covered in merge